### PR TITLE
Dialogue connections sometimes use the wrong offsets for the target endpoint

### PR DESCRIPTION
**Pull request by justinvdm**
Sometimes, connections seem to use a target endpoint's position for when the state is in edit mode, even if the state is in preview mode.
2015-04-29-150347_1440x900_scrot

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

**Comment by justinvdm** [comment]
@SteveBarnett erm, forgot to @ mention you in the ready-for-review ^

File: go/base/static/js/src/apps/dialogue/states/states.js [hunk 1]
```diff
@@ -390,6 +404,16 @@
     }
   });
 
+
+  function getEl(view) {
+    return view.$el;
+  }
+
+
+  function repaintEl(el) {
```
Comment (SteveBarnett) [inline go/base/static/js/src/apps/dialogue/states/states.js]: Empty function?

**Comment by SteveBarnett** [comment]
Left a tiny comment, otherwise looks good, :+1:

File: go/base/static/js/src/apps/dialogue/states/states.js (hunk 1 above)
Comment (justinvdm) [inline go/base/static/js/src/apps/dialogue/states/states.js]: Woops, thanks!

**Comment by justinvdm** [comment]
@SteveBarnett ready for re-review.

**Comment by SteveBarnett** [comment]
@justinvdm :+1:

**Pull request merged**